README.md

```diff
@@ -8,6 +8,15 @@ This repository will focus on the `DISCO UI` portion of this work.
 
 ![alt text](docs/diagrams/system_overview.png "Discovery system overview")
 
+## Environment variables
+
+### Required
+`VUE_APP_TIMDEX_API`: the root URL for the TIMDEX API
+
+### Optional
+`VUE_APP_RESULTS_PER_PAGE`: the number of records displayed per page
+`SENTRY_DSN`: set to Sentry project key to log exceptions
```

Comment:
Is this the right name? Or does it need to be prefixed with `VUE_APP_`?

Reply:
Thanks for catching this, @matt-bernhardt. Just pushed up the correction.